examples/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java

```diff
@@ -116,17 +118,13 @@ public void listFeatures(int lowLat, int lowLon, int hiLat, int hiLon) {
     Iterator<Feature> features;
     try {
       features = blockingStub.listFeatures(request);
+      for (int i = 1; features.hasNext(); i++) {
+        Feature feature = features.next();
+        info("Result #" + i + ": {0}", feature);
+      }
     } catch (StatusRuntimeException e) {
-      logger.log(Level.WARNING, "RPC failed: {0}", e.getStatus());
-      return;
-    }
-
-    StringBuilder responseLog = new StringBuilder("Result: ");
-    while (features.hasNext()) {
-      Feature feature = features.next();
-      responseLog.append(feature);
+      warning("RPC failed: {0}", e.getStatus());
```

Comment:
just being consistent with all other places in the class for error handling (printing a warning on the CLI prompt). why need propagate or other action?